cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -578,6 +578,7 @@ func (g *genConversion) generateConversion(inType, outType *types.Type, sw *gene
 		}
 	} else {
 		// Emit a public conversion function.
+		sw.Do("//"+nameTmpl+" ... \n", args)
```

Comment:
Can you add space after "//"? I mean:
```sw.Do("// "+nameTmpl+" ... \n", args) ```

Also, maybe change to slightly better comment (for doc purpose), e.g.:
s/... /is autogenerated conversion function/
or sth like that?